src/content/docs/ecosystem/validators/validator-cli/self-recharging-wallets.mdx

```diff
@@ -9,53 +9,53 @@ import { Aside } from "@astrojs/starlight/components";
-| nodeExit()     | Validator            | To remove a node from the network due to hardware failure or unmet MSR.                                                                              |
-| complaint()    | Validator            | Performed in DKG failure mode when a node lodges a general DKG complaint against a malicious node who may have failed to send data.                  |
-| complaintBadData() | Validator       | Performed in DKG failure mode when a node discovers that malicious data has been sent and requires verification data to validate.                     |
-| preResponse()  | Validator            | Performed in DKG failure mode, when a node sends a verification vector and secret key shares ahead of the final `response()` result.                 |
-| response()     | Validator            | Performed in DKG failure mode when a node sends a response to a `complaint()` to prove it's not malicious based on the data it sends in `preResponse()`. |
-| broadcast()    | SKALE Chain Owner    | Performed during SKALE Chain creation when distributed key generation process begins. Nodes broadcast their secret key contributions to all other nodes. |
-| alright()      | SKALE Chain Owner    | Performed during SKALE Chain creation when distributed key generation is in-process. Nodes send receipts of secret key contributions.                |
+| getBounty()    | Validator            | Called every month by each supernode to request bounty.                                                                                                   |
+| nodeExit()     | Validator            | To remove a supernode from the network due to hardware failure or unmet MSR.                                                                              |
+| complaint()    | Validator            | Performed in DKG failure mode when a supernode lodges a general DKG complaint against a malicious supernode who may have failed to send data.                  |
+| complaintBadData() | Validator       | Performed in DKG failure mode when a supernode discovers that malicious data has been sent and requires verification data to validate.                     |
+| preResponse()  | Validator            | Performed in DKG failure mode, when a supernode sends a verification vector and secret key shares ahead of the final `response()` result.                 |
+| response()     | Validator            | Performed in DKG failure mode when a supernode sends a response to a `complaint()` to prove it's not malicious based on the data it sends in `preResponse()`. |
+| broadcast()    | SKALE Chain Owner    | Performed during SKALE Chain creation when distributed key generation process begins. Supernodes broadcast their secret key contributions to all other supernodes. |
+| alright()      | SKALE Chain Owner    | Performed during SKALE Chain creation when distributed key generation is in-process. Supernodes send receipts of secret key contributions.                |
```

Comment:
@DmytroNazarenko Confirming are these functions done at the Supernode or Virtualized Node level?